public/services.ts

```diff
@@ -49,4 +60,5 @@ export default {
   getOverlays,
   setUISettings,
   setQueryService,
+  getSavedObjectsClient,
```

Comment:
nit: can we add the other added getters here? To make mocking easier for tests.

Reply:
Are you going to make this change?

Reply:
no, this change will be made in UT pr